### ISSUE TITLE
Reduce Citations block and output to only display what has values

### ISSUE THREAD

**Issue by wgilling** [asulibraries/islandora-repo#159]
EDIT: the Repo Design meeting on 9/25, we proposed to remove the block and page related to these and to merely display the relevant fields on the item page in the section that is below the Details section where the block and share block were set to be. This proposal would also remove the second "print & share" block on that part of the page.

This is an issue that came out of the design group meetings -- to remove any links to citation fields that have no data as well as the output in the items/{nid}/citations view.

**Comment by wgilling** [comment]
We get three fields from metadata:

Statement of responsibility - text (under responsibilities of use)
Copyright statement - link to taxonomy (taxonomy term has URL, label, and logo) (under licensing and permissions)
Reuse permissions - link to taxonomy (taxonomy term has URL, label, and logo) (under copies and reproductions)
Preferred citation - text (under cite this item section)
Should entire blocks (and connected links in the blocks) be removed when the field is not populated?
(from 9/18) - Inclined to display what we have
(9/22) - Want to run by Debra and Stacey

**Comment by elizoller** [comment]
Design decided we did not have enough content to full the whole page, so we've decided to remove the sidebar for citations, etc and remove the additional page (for now). the preferred citation with copy to clipboard feature will go in the main content on the item page under "interact with this item" section. also if in the rare chance there is a statement of responsibility, it would go there as well.
